The tests using tmux are inherently racy as we are sending keystrokes to the
terminal without knowing at which point they have been fully processed by the
application holding the control terminal.

Therefore, the fish tests using an opportunistic sleep here. Unfortunately, it
seems that sleeping for 1 second is not enough on some (slower) architectures.
For example, presently, some tmux-related tests fail on ppc64le (only on the
builders, not on the CI):

	The following tests FAILED:
		125 - tmux-abbr.fish (Failed)
		127 - tmux-bind.fish (Failed)
		129 - tmux-commandline.fish (Failed)
		134 - tmux-multiline-prompt.fish (Failed)

This patch attempts to workaround that by increasing the sleep interval. If
these tests continue to fail in the future, we could just disable them due to
their inherently racy nature.

--- fish-4.0.0.orig/tests/test_functions/isolated-tmux-start.fish
+++ fish-4.0.0/tests/test_functions/isolated-tmux-start.fish
@@ -28,7 +28,7 @@
     end
 
     function tmux-sleep
-        set -q CI && sleep 1
+        set -q CI && sleep 3
         or sleep 0.3
     end
 
